doc/building.md

```diff
@@ -4,37 +4,53 @@ This guide documents the Onyx build process. For example usage of this building
 
 Note that steps 0, 1 and 2 can be skipped if you have already done them before.
 
-## Step 0: Set up the build environment
+## Step 0: Get required packages
 
-First, you'll need to download the [minimal sysroot](https://storage.googleapis.com/onyx-stuffs/minimal-sysroot.tar.zst).
-This allows you to have the toolchain build against a sane libc.
-Be sure to export `SYSROOT=$PWD/sysroot` and `ONYX_ARCH=x86_64` (right now, only x86_64 is supported).
+You need the following packages and programs: mtools, xorriso, libfl2, ninja, gn, grub-mkrescue.
+Also you need development files for MPC, MPFR and GMP and texinfo package if you are going to build toolchains.
 
-## Step 1: Get the toolchains
+## Step 1: Fetch submodules
 
```

Comment:
I didn't have this step because I usually do git clone <url> --recursive, may be worth mentioning that